sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -920,6 +926,18 @@ public CaseDataDto postUpdate(String uuid, JsonNode caseDataDtoJson) {
 				eventSummaries = eventService.getEventSummaryDetailsByCases(resultCaseIds);
 			}
 
+			Map<Long, UserReference> caseUsers = null;
```

Comment:
I think it is generally safer to avoid use of nulls. It would also be ok to create a new HashMap and check for isEmpy() later.

One better approach is to have a separate method:
`public Map<Long, UserReference> getCaseUsers(ExportConfigurationDto exportConfiguration) {
if(exportConfiguration blabla) {
// create the map and return it
} else {
return Collections.emptyMap();
}
}`

`
Map<Long, UserReference> caseUsers = getCaseUsers(exportConfiguration);
`